libs/ts-rest/nest/src/lib/ts-rest.interceptor.ts

```diff
@@ -0,0 +1,65 @@
+import {
+  applyDecorators,
+  CallHandler,
+  Delete,
+  ExecutionContext,
+  Get,
+  Injectable,
+  NestInterceptor,
+  Patch,
+  Post,
+  Put,
+  SetMetadata,
+  UseInterceptors,
+} from '@nestjs/common';
+import { map, Observable } from 'rxjs';
+import { Response } from 'express-serve-static-core';
+import { AppRoute } from '@ts-rest/core';
+import { tsRestAppRouteMetadataKey } from './typed-request.decorator';
+
+@Injectable()
+export class TsRestInterceptor implements NestInterceptor {
```

Comment:
Exactly what I was saying, `TsRestInterceptor` makes me _know_ that I'm using ts-rest. I love this personally.